build/balena-build.sh

```diff
@@ -17,6 +17,7 @@ Usage: ${script_name} [OPTIONS]
     -g Barys extra arguments (optional, for example '-g "-a VARIABLE=value"')
     -a Balena API environment (defaults to "balena-cloud.com")
     -t Balena API token (optional - private apps access)
+    -k Keep local containers (optional- by default container iamges are removed)
```

Comment:
```suggestion
    -k Keep local containers (optional - by default container iamges are removed)
```
Smallest nit of all time, please ignore